### DISCUSSION TITLE
@SirGiraffey's Assets

### DISCUSSION THREAD

**Discussion by zicklag** [fishfolk/jumpy#851] [Assets]
Here's a place for you to upload your assets whenever you have new ones @SirGiraffey.

**Comment by SirGiraffey** [comment]
Awesome, here are the icon files (in PNG format):
![PlayerIcon16x16](https://github.com/fishfolk/jumpy/assets/57463655/38f13414-c683-4b86-b427-46b06c32d792)
![PlayerIcon32x32](https://github.com/fishfolk/jumpy/assets/57463655/4894b7f6-e606-4756-b858-c3e2f016a2cf)
![PlayerIcon64x64](https://github.com/fishfolk/jumpy/assets/57463655/231daa6b-15bb-45d3-a5f7-57f5ca6e05e5)
![PlayerIcon96x96](https://github.com/fishfolk/jumpy/assets/57463655/192f0d62-5965-4f01-b5bd-e37f13166321)

And here are profile avatar icons for Steam users:
![Avatar1](https://github.com/fishfolk/jumpy/assets/57463655/2adde1e8-e189-492d-9e2f-aa095551d3a9)
![Avatar2](https://github.com/fishfolk/jumpy/assets/57463655/4ac12da2-275b-475a-b52f-e2e9e707b7bd)

**Reply by SirGiraffey** [reply]
Here's a closeup for the smallest one :D 
![obraz](https://github.com/fishfolk/jumpy/assets/57463655/c31375ca-a210-4141-b7e1-1d7f871db62d)
